app/robots/iCubGenova04/leftHandRetargetingParams.ini

```diff
@@ -1 +1,14 @@
 scalingFactor           0.63
+
+# additional rotations
+# the following rotation map the hand oculus frame and the hand robot frame (Identity)
+handOculusFrame_R_handRobotFrame ((1.0 0.0 0.0), (0.0 1.0 0.0), (0.0 0.0 1.0))
+
+# the following rotation map the teleoperation frame and the teleoperation robot
+# frame in this case the teleoperation robot frame chosen is "imu_frame" and
+# according to icub cad has:
+# The z-axis is parallel to gravity but pointing upwards.
+# The x-axis points behind the robot.
+# The y-axis points laterally and is chosen according to the right-hand rule.
+# So the rotation matrix is simply Rotz(pi)
+teleoperationRobotFrame_R_teleoperationFrame ((-1.0 0.0 0.0), (0.0 -1.0 0.0), (0.0 0.0 1.0))
```

Comment:
Even if it can be devised from the above comments, it may be worthwhile to specify how the teleoperation frame is oriented.

Reply:
Done in a19e44d